iothub/device/src/InternalClient.cs

```diff
@@ -473,9 +518,14 @@ public async Task RejectMessageAsync(Message message, CancellationToken cancella
         }
 
         /// <summary>
-        /// Retrieve the device twin properties for the current device.
-        /// For the complete device twin object, use Microsoft.Azure.Devices.RegistryManager.GetTwinAsync(string deviceId).
+        /// Retrieve the twin properties for the current client. The client instance must be opened already.
+        /// For the complete twin object, use Microsoft.Azure.Devices.IotHubServiceClient.Twins.GetAsync(string deviceId, CancellationToken cancellationToken)
```

Comment:
It feels odd to recommend users to call a service client API here in the device client docs. Maybe we need more nuance saying that the device-side view of the twin is different from the service-side view and link to some msdoc explaining further?

Reply:
How about this:
```c#
/// <summary>
/// Retrieve the twin properties for the current client. The client instance must be opened already.
/// This API gives you the client's view of the twin. For more information on twins in IoT hub see <see href="https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-device-twins"/>.
/// </summary>
```

Reply:
Suggest putting the second sentence in <remarks>.